GLOBALS.md

```diff
@@ -38,3 +38,11 @@ Spec: https://www.w3.org/TR/wasm-js-api-1/
 Environments supported:
 - [Web](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly)
 - Node.js: not documented yet
+
+### Console API
+
+Spec: https://console.spec.whatwg.org
+
+Environments supported:
+- Web
```

Comment:
MDN page doesn't exist yet. Do we have some other good documentation websites?

Reply:
There is https://developer.mozilla.org/en-US/docs/Web/API/console